dali-mqtt-daemon.py

```diff
@@ -289,13 +291,35 @@ def create_mqtt_client(
     }
 
     exception_raised = False
+    all_supported_log_levels = {
+        "critical": logging.CRITICAL,
+        "error": logging.ERROR,
+        "warning": logging.WARNING,
+        "info": logging.INFO,
+        "debug": logging.DEBUG,
+    }
+    args.log_level = args.log_level.lower()
+    if args.log_level not in all_supported_log_levels:
+        logger.error(
+            "Unsupported log level {}! Changed to level info".format(args.log_level)
+        )
+        args.log_level = "info"
+    logger.setLevel(all_supported_log_levels[args.log_level])
     try:
         dali_driver = None
         logger.debug("Using <%s> driver", args.dali_driver)
         if args.dali_driver == HASSEB:
             from dali.driver.hasseb import SyncHassebDALIUSBDriver
 
             dali_driver = SyncHassebDALIUSBDriver()
+            firmware_version = float(dali_driver.readFirmwareVersion())
+            if firmware_version < MIN_HASSEB_FIRMWARE_VERSION:
+                logger.error("Using a script requires newest hasseb firmware")
+                logger.error(
+                    "Please, look at https://github.com/hasseb/python-dali/tree/3dbf4af3b3770431e7351057ea328b4dbcc3a355/dali/driver/hasseb_firmware"
```

Comment:
this url points to a very specific commit (not a good idea)

We should get in touch with hasseb and propose to move this information into his fork Wiki